client-admin/package.json

```diff
@@ -4,60 +4,56 @@
   "repository": "github:pol-is/polisClientAdmin",
   "description": "Polis Admin Console",
   "scripts": {
-    "clean": "rimraf dist",
-    "build:webpack": "npm run clean && NODE_ENV=production webpack --config webpack.config.js",
-    "build:webpack_unminified": "npm run clean && NODE_ENV=development webpack --config webpack.config.unminified.js",
-    "build": "npm run build:webpack",
-    "build-debug": "npm run build:webpack_unminified",
-    "deploy:prod": "node gulpfile.js deploy_TO_PRODUCTION",
```

Comment:
I did a test merge of this into the compdemocracy/dev branch, but I ran into this error when trying build with `docker compose` due to the removal of `deploy:prod`:
```
Step 11/42 : RUN npm run deploy:prod
 ---> Running in e2442f4e8746
npm ERR! missing script: deploy:prod
npm ERR! 
npm ERR! Did you mean this?
npm ERR!     build:prod
```
I see that client-admin/Dockerfile has been updated to use `build:prod` instead of `deploy:prod`, but it looks like when I am building it must be running either Dockerfile or file-server/Dockerfile, which both use ` deploy:prod` and still reference node:11.15.0-alpine. 

From looking at the docker-compose.yml file I think that it might be the file-server/Dockerfile that is making client-admin instead of client-admin/Dockerfile. I might just be doing something wrong when I build it, but it still might be good to remove gulp from the sections of Dockerfile and file-server/Dockerfile that relate to building client-admin so that they are all consistent

Reply:
The problem is that there is some duplication of efforts in the client build process, as you note, between the `file-server/Dockerfile` and the individual client Dockerfiles. We duplicated this work into the `client-server/Dockerfile` to avoid issues with Dockerfiles referencing each other in certain build environments (Heroku, podman). I think we want to try and consolidate these if possible, potentially via multi-stage builds, but we shouldn't hang this PR up on that. We _should_ however continue to keep all of the Dockerfiles up to date, and switch everything over to `build:prod` (vs `deploy:prod`). So, thanks for catching this.